utils.go

```diff
@@ -158,16 +159,27 @@ func getTomlStorage(storeOptions *StoreOptions) *tomlConfig {
 	return config
 }
 
+func getRootlessUID() int {
+	uidEnv := os.Getenv("_CONTAINERS_ROOTLESS_UID")
+	if uidEnv != "" {
+		u, _ := strconv.Atoi(uidEnv)
+		return u
+	}
+	return os.Geteuid()
+}
+
 // DefaultStoreOptions returns the default storage ops for containers
-func DefaultStoreOptions(rootless bool, rootlessUid int) (StoreOptions, error) {
+func DefaultStoreOptions() (StoreOptions, error) {
```

Comment:
@giuseppe, would you open a PR at libpod for this change? Just yesterday I was cleaning up the API breakage there and this will require changes over a libpod to make it build.

Reply:
this is going to break containers/image as well :-)

Reply:
How about you leave DefaultStoreOptions alone, and add DefaultStoreOptionsFromEnv()